config/locales/en.yml

```diff
@@ -104,3 +104,20 @@ en:
     - image: 'img-connect.svg'
       subtitle: 'Connect'
       description: You're not alone. Learn and get help from our friendly community of beginner and experienced developers.
+  contributing_page_beliefs:
+    - short_text: Flexibility
```

Comment:
Nit: What's your thoughts about changing all `short_text` instances between this and the about page beliefs to `title` instead? 

I know you got the `short_text` naming from the about page beliefs to keep the naming consistent and it's certainly not a blocker. I just noticed myself stoping to think about what this attribute represented.

Reply:
Good idea :bulb:, I think that reads much nicer